bundles/org.eclipse.swt/Eclipse SWT/gtk/org/eclipse/swt/widgets/Scrollable.java

```diff
@@ -303,6 +303,11 @@ public ScrollBar getHorizontalBar () {
  */
 public int getScrollbarsMode () {
 	checkWidget();
+	if (!GTK.GTK4) {
```

Comment:
Is this different on GTK4?

Reply:
On GTK4 GTK_OVERLAY_SCROLLING variable is not supported anymore. For reference see [GTK3 environment variables:](https://docs.gtk.org/gtk3/running.html#environment-variables) and [GTK4 environment variables](https://docs.gtk.org/gtk4/running.html#environment-variables) where GTK_OVERLAY_SCROLLING variable is not listed anymore.

Reply:
thanks